Patient of doctor

Thommond opened the pull request:
- Created the patientOfDoctor.php file to handle the information of patients.

- Updated doctorAppts.php to properly display the doctor of the daily roster.

- Created function to prevent the refresh of a PHP form so that URL id handling in patient doctor would not be interfered with and nor would the display of a message.

Thommond committed:
Updated the structure of the project so that there is a directory of doctor info pages.

Updated login.php to fit changes of directories.

Created patientOfDoctor.php file which identifies unique users based of 
id variable.

Thommond committed:
Updated patientOfDoctor to have back button to doctorpage.

Updated so that all previous appointments the patient had with this 
doctor will appear on patientOfDoctor.php.

Updated so that creating a new prescription is only possible if today is 
appt_day

Thommond committed:
Updated doctorAppt.php so that it properly fetched dotcor from roster table.

Thommond committed:
Created a function in register.js so that patientOfDoctor.php form would not refresh page since it would interfere with url id handling.

Updated patientOfDoctor.php so it checks if prescription has been made 
and if not then allows, if so it does not.

BrendanHorst commented on views/doctor/patientOfDoctor.php:
```diff
@@ -0,0 +1,172 @@
+      die('Error could not get desired patient info:' . mysqli_connect_error());
+    }
+
+    $sql = "SELECT  a.day, p.morning_med, p.afternoon_med, p.night_med, p.comment
+           FROM appointments as a JOIN prescriptions as p on (a.patient_id = p.patient_id) AND (a.day = p.appt_day)
+           JOIN users as u ON (a.patient_id=u.id)
+           WHERE doctor_id LIKE $id AND a.patient_id LIKE '$patient'";
+
+
+           $result = mysqli_query($db_link, $sql);
+
+           if (empty($result)) {
+             die("<p class='error'>Patient info has no results</p>");
+           }
```
Most of the rest of the file is indented from here; it should be in line with $sql

BrendanHorst commented on static/register.js:
```diff
@@ -13,3 +13,9 @@ select.addEventListener('change', (event) => {
   }
 })
+
+// For patientOfDoctor preventing refresh
+let form = document.getElementById("pre_form");
+
+function handleForm(event) { event.preventDefault(); }
+form.addEventListener('submit', handleForm);
```
I think this would make sense to be in a different file specific to the patientOfDoctor page to prevent errors

BrendanHorst commented on views/doctor/patientOfDoctor.php:
```diff
@@ -0,0 +1,172 @@
+
+            if (isset($_POST['submit'])) {
+
+              $day = date('Y-m-d');
+              $Comment = $_POST['comment'];
+              $Mmed = $_POST['morning'];
+              $Amed = $_POST['afternoon'];
+              $Nmed = $_POST['night'];
```
Semantically, I believe these variables should all be lowercase.

Thommond committed:
Updated the js for patientOfDoctor.php to make semantic sense.

Updated the case of comments in patientOfDoctor.php

Updated the overall indentation of patientOfDoctor.php
